api/app/sfms/raster_addresser.py

```diff
@@ -81,6 +99,16 @@ def get_calculated_precip_key(self, datetime_to_calculate_utc: datetime):
         calculated_weather_prefix = f"{self.weather_model_prefix}/{datetime_to_calculate_utc.date().isoformat()}/"
         return os.path.join(calculated_weather_prefix, compose_computed_precip_rdps_key(datetime_to_calculate_utc))
 
+    def get_daily_ffmc(self, timestamp: datetime, run_type: RunTypeEnum):
+        assert_all_utc(timestamp)
+        iso_date = timestamp.date().isoformat()
+        return f"sfms/uploads/{run_type.value}/{iso_date}/fine_fuel_moisture_code{iso_date.replace('-', '')}.tif"
+
+    def get_calculated_daily_ffmc(self, timestamp: datetime):
```

Comment:
I was using the generic `get_calculated_index_key` for this

Reply:
Similar reason to: https://github.com/bcgov/wps/pull/4081/files#r1838825805